.github/workflows/external-filter.yml

```diff
@@ -0,0 +1,17 @@
+name: External Trigger Filter
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    branches:
+      - 'master'
```

Comment:
Only PRs (from forks) opened against `master` will be considered. All others will be skipped automatically.